common/constants.js

```diff
@@ -35,10 +38,14 @@ module.exports = {
   NEWTAB_PREFS_ENCODING: {
     "showSearch": 1 << 0,
     "showTopSites": 1 << 1,
-    "showHighlights": 1 << 2,
-    "showMoreTopSites": 1 << 3
+    "showPocketStories": 1 << 2,
```

Comment:
Please append the new item to the end this encoding table other than inserting it in the middle. Otherwise, it'll break the existing telemetry.

i.e. 
```
NEWTAB_PREFS_ENCODING: {
     "showSearch": 1 << 0,
     "showTopSites": 1 << 1,
     "showHighlights": 1 << 2,
    "showMoreTopSites": 1 << 3,
    "showPocketStories": 1 << 5
}
```